onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -1004,6 +1004,10 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
               "Custom scale will be used if specified. Default value is 1/sqrt(head_size)",
               AttributeProto::FLOAT,
               OPTIONAL_VALUE)
+        .Attr("local_window_size",
+              "left_window_size for local attention (like Mistral). Default value is -1 meaning unused.",
```

Comment:
what's the meaning of value 0 and 1?

Reply:
Shall we have two attributes like `left_window_size`, `right_window_size` according to flash API?